config/jobs/kubernetes-sigs/cluster-capacity/cluster-capacity-presubmits-release-1.27.yaml

```diff
@@ -1,52 +1,77 @@
 # sigs.k8s.io/cluster-capacity presubmits
 presubmits:
   kubernetes-sigs/cluster-capacity:
-  - name: pull-cluster-capacity-verify-gofmt-release-1-22
+  - name: pull-cluster-capacity-verify-gofmt-release-1-27
+    cluster: eks-prow-build-cluster
     decorate: true
     path_alias: sigs.k8s.io/cluster-capacity
     branches:
     # The script this job runs is not in all branches.
-    - ^release-1.22$
+    - ^release-1.27$
     always_run: true
     spec:
       containers:
-      - image: public.ecr.aws/docker/library/golang:1.16.14
+      - image: public.ecr.aws/docker/library/golang:1.19.3
```

Comment:
1.20.* based on https://github.com/kubernetes/kubernetes/blob/release-1.27/go.mod#L9

Reply:
Thanks! I was basing this on the descheduler jobs. I'll update 1.27 for both.

Reply:
Nevermind, descheduler jobs look ok.